lte/gateway/c/session_manager/MeteringReporter.cpp

```diff
@@ -53,6 +53,13 @@ void MeteringReporter::report_usage(
   report_download(imsi, session_id, total_rx);
 }
 
+void MeteringReporter::initialize_usage(
+    const std::string& imsi, const std::string& session_id,
+    SessionCredit::TotalCreditUsage usage) {
+  report_upload(imsi, session_id, usage.monitoring_tx + usage.charging_tx);
+  report_download(imsi, session_id, usage.monitoring_rx + usage.charging_rx);
+}
+
```

Comment:
what will happen if the subscriber was not using any data before the restart. Will reporting the same value we had already reported be a problem?

Reply:
The counter always expects the cumulative value, so I don't think we need to worry about duplicate reporting here.